modules/i18n/src/translations/ta-IN/portals/common.ts

```diff
@@ -115,6 +115,7 @@ export const common: CommonNS = {
     showLess: "சுருக்கிக் காண்",
     showMore: "மேலுங் காண்",
     showPassword: "கடவுச்சொல்லை காண்",
+    generatePassword: "Generate password",
```

Comment:
@thivi Please validate.

```suggestion
    generatePassword: "கடவுச்சொல்லை உருவாக்கவும்",
```